spec/rubocop/cop/layout/empty_lines_around_attribute_accessor_spec.rb

```diff
@@ -57,6 +66,17 @@ def do_something
     RUBY
   end
 
+  it 'accepts code that separates attribute accessors from the code and comment line with a newline' do
+    expect_no_offenses(<<~RUBY)
+      attr_accessor :foo
+      attr_reader :bar
+      attr_writer :baz
+      # comment
```

Comment:
Ditto.

Reply:
Indeed, I've corrected this one as well.